hello_world/transactions/package.json

```diff
@@ -8,5 +8,12 @@
   },
   "keywords": [],
   "author": "",
-  "license": "ISC"
+  "license": "ISC",
```

Comment:
Since this is public repo, I think the package.json format should follow the standard

Reply:
What is the standard here?

Reply:
mostly it's like https://github.com/LiskHQ/lisk-sdk/blob/development/sdk/package.json